crates/sui-types/src/messages_grpc.rs

```diff
@@ -219,6 +219,16 @@ pub struct HandleCertificateRequestV3 {
     pub include_auxiliary_data: bool,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct HandleTransactionRequestV2 {
```

Comment:
I wonder if we should give it a new schema of names.
Having HandleTransaction in the request type but HandleCertificate in the response type is confusing to me.

Reply:
I can make a new response type, was just trying to save time thinking the output would be the same, but clearly that's wrong anyway if as Mingwei says we won't be able to sign the effects